tests/core/cgo/asm_dep_conditional_cgo/asm_dep_conditional_cgo.go

```diff
@@ -0,0 +1,14 @@
+package main
```

Comment:
Is this file used anywhere?

Reply:
Oops thanks, no. Removed this and the other main program. I used them to understand/debug what was happening, then converted them into the test version.